src/map-marker/__tests__/tile-grid.js

```diff
@@ -53,21 +53,14 @@ const TileGrid = ({children, customizerOptions, cols}: TileGridPropsT) => {
               >
                 <LabelSmall color={contentSecondary}>{label}</LabelSmall>
```

Comment:
Block and Label are duplicated above and below. Instead, `Label3` had to be changed to `LabelSmall`.